core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -123,6 +126,7 @@ private[spark] class SparkSubmitArguments(args: Seq[String], env: Map[String, St
       .orNull
     name = Option(name).orElse(sparkProperties.get("spark.app.name")).orNull
     jars = Option(jars).orElse(sparkProperties.get("spark.jars")).orNull
+    ivyRepoPath = sparkProperties.get("spark.jars.ivy").orNull
```

Comment:
Actually in order to not expose it to users in spark-submit. I still wanted to have it as a configuration just for the flexibility.